appveyor.yml

```diff
@@ -9,12 +9,13 @@ init:
 
 environment:
   matrix:
-    # node.js
+    - nodejs_version: 8
     - nodejs_version: 10
+    - nodejs_version: 12
 
 install:
-  - ps: Install-Product node $env:nodejs_version
-  - npm install
+  - ps: Update-NodeJsInstallation (Get-NodeJsLatestBuild $env:nodejs_version)
+  - npm ci || npm install
```

Comment:
I guess this `|| npm install` fallback is not needed, because [npm CLI update was backported to Node.js 8](https://github.com/nodejs/node/blob/1c2616971417bee811ea00da436c87a489f9b1ed/doc/changelogs/CHANGELOG_V8.md#2018-09-11-version-8120-carbon-lts-mylesborins).

Reply:
TIL!